Jon-Jonrosario5/Week-8/state-capitals/capitals.py

```diff
@@ -167,6 +168,7 @@
 
 prompt = '> '
```

Comment:
Not sure why this is here. Reasoning?

Reply:
It is more for a user friendly experience in the terminal. When prompted to answer a question, the ">" appears, it's kinda like a signal for the user to type something. A small add that wasn't necessary but a nice "Jon" touch.

Reply:
@DaltonHart